helm/azure-collector/templates/service.yaml

```diff
@@ -5,11 +5,10 @@ metadata:
   namespace: {{ tpl .Values.resource.default.namespace  . }}
   labels:
     {{- include "azure-collector.labels" . | nindent 4 }}
-  annotations:
-    prometheus.io/scrape: "true"
```

Comment:
Can we wrap this in a negative capabilities check so if the prom CRDs aren't available it still scrapes by annotation?

Reply:
We can but our prometheus setup is not using that annotation

Reply:
Right, ignore it then :)